src/npc.h

```diff
@@ -742,20 +741,25 @@ class npc : public player
      * This does not change the global position of the NPC.
      */
     tripoint global_square_location() const override;
- int plx, ply, plt;// Where we last saw the player, timeout to forgetting
- int itx, ity; // The square containing an item we want
- int guardx, guardy;  // These are the local coordinates that a guard will return to inside of their goal tripoint
+    tripoint last_player_seen_pos; // Where we last saw the player
+    int last_seen_player_turn; // Timeout to forgetting
```

Comment:
So is it a timeout or a turn number? The name indicates turn, but the comment says timeout, that is a bit confusing.

But it doesn't matter anyway because it's not used, only set to 999 in the constructor and later only read, never again changed at all.